HACKING.md

```diff
@@ -138,55 +140,87 @@ If a test hangs, you can enable verbose mode:
 
 (or -check.v for less verbose output).
 
-There is more to read about the testing framework on the [website](https://labix.org/gocheck)
+There is more to read about the testing framework on the 
+[website](https://labix.org/gocheck)
 
-### Running the spread tests
+### Running spread tests
 
-To run the spread tests locally you need the latest version of spread
-from https://github.com/snapcore/spread. It can be installed via:
+To run the spread tests locally via QEMU, you need the latest version of
+[spread](https://github.com/snapcore/spread). You can get spread, QEMU, and the
+build tools to build QEMU images with:
 
-    $ sudo apt install qemu-kvm autopkgtest
-    $ sudo snap install --devmode spread
+    $ sudo apt update && sudo apt install -y qemu-kvm autopkgtest
+    $ curl https://niemeyer.s3.amazonaws.com/spread-amd64.tar.gz | tar -xz -C $GOPATH/bin
 
-Then setup the environment via:
+#### Building spread VM images
 
-    $ mkdir -p .spread/qemu
-    $ cd .spread/qemu
-    # For xenial (same works for yakkety/zesty)
-    $ adt-buildvm-ubuntu-cloud -r xenial
-    $ mv adt-xenial-amd64-cloud.img ubuntu-16.04.img
-    # For trusty
-    $ adt-buildvm-ubuntu-cloud -r trusty --post-command='sudo apt-get install -y --install-recommends linux-generic-lts-xenial && update-grub'
+To run the spread tests via QEMU you need to create VM images in the
+`~/.spread/qemu` directory:
+
+    $ mkdir -p ~/.spread/qemu
+    $ cd ~/.spread/qemu
+
+Assuming you are building on an Ubuntu 18.04 Bionic Beaver (or a later 
+development release like Ubuntu 19.04 Disco Dingo), run the following to build a
+64-bit Ubuntu 16.04 VM to run the spread tests on:
+
+    $ autopkgtest-buildvm-ubuntu-cloud -r xenial
+    $ mv autopkgtest-xenial-amd64.img ubuntu-16.04-64.img
+
+To build a Ubuntu 15.04 Trusty Tahr based VM, use:
```

Comment:
To build *an* Ubuntu *14.04* LTS (Trust Tahr)...

Reply:
Fixed